x-pack/plugins/synthetics/public/apps/synthetics/components/test_run_details/components/console_icon.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as React from 'react';
+import {
+  EuiButtonIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCodeBlock,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useRef, useState } from 'react';
+
+export const ConsoleInfoIcon = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const timer = useRef<NodeJS.Timeout | null>(null);
```

Comment:
We're trying to ensure that the user doesn't accidentally activate the popover?

We could reduce significant complexity by only displaying the popover on `click` or `key down`, vs. also on `mouse over`, or just popping it open when the mouse enters, and bubbling `mouse leave` event, similar to what we do on the overview page cards.